.github/workflows/release.yml

```diff
@@ -46,13 +61,15 @@ jobs:
           /bin/bash arbitrary-users-patch/build_images.sh --push
 
       - name: Build and push images
-        run: |
-          SHORT_SHA1=$(git rev-parse --short HEAD)
-          VERSION=$(head -n 1 VERSION)
-          IMAGE=che-devfile-registry
-          DOCKERFILE_PATH=./build/dockerfiles/Dockerfile
-          docker build -t ${IMAGE} -f ${DOCKERFILE_PATH} --build-arg PATCHED_IMAGES_TAG=${VERSION} --target registry .
-          docker tag ${IMAGE} quay.io/eclipse/${IMAGE}:${SHORT_SHA1}
-          docker push quay.io/eclipse/${IMAGE}:${SHORT_SHA1}
-          docker tag ${IMAGE} quay.io/eclipse/${IMAGE}:${VERSION}
-          docker push quay.io/eclipse/${IMAGE}:${VERSION}
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./build/dockerfiles/Dockerfile
+          target: registry
+          build-args: |
+            PATCHED_IMAGES_TAG=${{ steps.prep.outputs.version }}
+          platforms: linux/amd64,linux/s390x
+          tags: quay.io/eclipse/${IMAGE}:${{ steps.prep.outputs.version }},quay.io/eclipse/${IMAGE}:${{ steps.prep.outputs.short_sha1 }}
```

Comment:
`${IMAGE}` should be `che-devfile-registry`. Currently it will evaluate to nothing.

Reply:
That was a miss from my side. Implemented the change, please review.